ross/bearing_seal_element.py

```diff
@@ -109,6 +111,35 @@ class BearingElement(Element):
         Array with the speeds (rad/s).
     Examples
     --------
+    >>> # A bearing element located in the first rotor node, with these
+    >>> # following stiffness and damping coefficients and speed range from
+    >>> # 0 to 200 rad/s
+    >>> Kxx = 1e6
+    >>> Kyy = 0.8e6
+    >>> Cxx = 2e2
+    >>> Cyy = 1.5e2
+    >>> W = np.linspace(0,200,11)
+    >>> bearing0 = BearingElement(n=0, kxx=Kxx, kyy=Kyy, cxx=Cxx, cyy=Cyy, w=W)
+    >>> bearing0.K(W)
+    array([[[1000000., 1000000., 1000000., 1000000., 1000000., 1000000.,
```

Comment:
I recommend using `# doctest: +ELLIPSIS` on the docstring examples where we have a large size array as output.
Another option is to slice the output.
This way we avoid having a large array cluttering the docstring.